sim/shaman/elemental/TestElemental.results

```diff
@@ -1012,15 +1012,15 @@ dps_results: {
 dps_results: {
  key: "TestElemental-AllItems-MandalaofStirringPatterns-62467"
  value: {
-  dps: 34363.35035
-  tps: 597.13505
+  dps: 34203.82708
+  tps: 596.51235
  }
 }
 dps_results: {
  key: "TestElemental-AllItems-MandalaofStirringPatterns-62472"
  value: {
-  dps: 34363.35035
-  tps: 597.13505
+  dps: 34203.82708
```

Comment:
Interesting that a couple items went down, will try to double check what happened. The ones that went up should mostly be low-ilvl or healer-stat ones that I excluded from the original list to keep it a bit shorter.

Reply:
Oh right it's just RNG having different outcomes since the cast order changed, disregard